### PR TITLE
Add style when the mark is empty and the mouse is over the sqare.

### PR DESCRIPTION
**Pull request by JoseManuelCaballero**

### CODE REVIEW

File: src/components/Board.jsx [hunk 1]
```diff
@@ -2,15 +2,17 @@ import React from "react";
 import PropTypes from "prop-types";
 import Square from "./Square";
 
-const Board = ({ board: squares, onPlay }) => {
+const Board = ({ board: squares, onPlay, nextPlayer }) => {
   return (
     <div className="board">
+      {console.log("next player",nextPlayer)}
```
Comment (bcisneros) [inline src/components/Board.jsx]: Remove this console.log

**Review by bcisneros — CHANGES_REQUESTED** [review]

**Comment by JoseManuelCaballero** [comment]
In the following image we can see a demonstration of the changes.

image

**Review by bcisneros — APPROVED** [review]
Thanks @JoseManuelCaballero for the effort. 

I will approve this PR but I think your approach has some disadvantages. For example, if we need to change either the background color or the font color, those elements will need to be updated together. Another disadvantage is that we are putting the value inside the square element, so if we recover the html it will be displayed incorrectly.

Beside that I think you did a great job 👏 !

**Comment by JoseManuelCaballero** [comment]
Thank you very much for your feedback!

**Pull request merged**